backend/infrastructure/mongo/client_handler.py

```diff
@@ -0,0 +1,34 @@
+import gridfs
+from pymongo import MongoClient
+from typing import Optional
+from bson import ObjectId
+from fastapi import HTTPException
+
+client = MongoClient("mongodb://localhost:27017/")
+db = client["client"]
+fs = gridfs.GridFS(db)
+
+def register_client(email: str, payment_address: str, delivery_address:str, nip: str, orders:str, password: str, company_name: str) -> str:
+
```

Comment:
Business rules should not be  a part of infrastrructure package. Infrusturce should depend on business rules not the other way